config/qualifications.yml

```diff
@@ -21,10 +21,11 @@ title: Read the transportation programmer policies
 description: >
   This document lives in our programmer training [box folder](https://umass.app.box.com/folder/77237337598)
 ---
-title: Join Slack
+title: Join Teams
 description: >
-  We use [Slack](https://slack.com/) for communication every day. After your invitation to our
-  Slack team, you can download the desktop version or use the web version, whichever suits you.
+  We use [Microsoft Teams](https://www.umass.edu/it/disruptionresilience/communication-collaboration#Communicate%20with%20Teams%20&%20Peers)
+  for communication every day. We will invite you to our team, after which you can
```

Comment:
Yeah, is that the workflow? Is there an "invite"?

https://aka.ms/mstfw seems to be a pretty stable link.  They should log in with their `-af` account when prompted.

Desktop client:

![2020-04-22_14-50](https://user-images.githubusercontent.com/240275/80021620-c607e600-84a8-11ea-97e4-002d2451b88f.png)

Reply:
No, I guess there's no invite, one just gets added. I'll change the wording and make sure to mention the -AF account thing.

Reply:
The wording I came up with is a shitshow.

Reply:
Two thoughts:

1.  Maybe its worth metioning the "you have two accounts" thing earlier? It's not really Teams-specific.
2.  Counterpoint: it's one of those things where the Trainee doesn't really have to take any action other than "wait for Adam to do a thing".

Reply:
Leaving out the "you have two accounts" (except for mentioning it here where they must log in as that separate account) was deliberate on my part because of that counterpoint. We had previously talked about how there were a lot of "qualifications" where it was simply "wait for your Adam to do a thing" and I removed them.
So if I'm going to mention it, I should mention all the other things as well that are "wait for Adam/Karin to do a thing" which would be a reversal of that earlier decision to not do that. Can we decide either way once and for all?

Reply:
I'm leaning towards not explaining the two yous in here. But a quick parenthetical here would be fine:

> We use [Microsoft Teams](https://aka.ms/mstfw) for communication every day. Log into teams with your "-af" account.